src/stretchy.js

```diff
@@ -227,13 +234,12 @@ const svgSpan = function(group, options) {
             } else {
                 span = buildCommon.makeSpan([widthClass], [svgNode], options);
                 span.style.height = height + "em";
-                spans.push(span);
             }
+            spans.push(span);
         }
 
-        if (numSvgChildren > 1) {
-            span = buildCommon.makeSpan(["stretchy"], spans, options);
-        }
+        span = numSvgChildren === 1 ? spans[0] :
+            buildCommon.makeSpan(["stretchy"], spans, options);
```

Comment:
This is to convince the compiler that `span` is indeed set. Also, unlike before this change, `span` is set also for `(numSvgChildren === 0)`. Unsure if that ever happens.

Reply:
We should probably be checking if `span` is defined below instead.  It's possible that we've messed up somewhere and the SVG path we're looking for doesn't exist.  We should probably throw on that and ask people to report it as a bug.

Reply:
`numSvgChildren` is always 1, 2, or 3. If I get creative, may I can think of reasons for `numSvgChildren` > 3 but I can't think of any reasons for `numSvgChildren` = 0.

Reply:
`numSvgChildren > 3` would mean that `widthClass` would be `undefined`. Is that okay?
For now, I've updated the code with an error requiring 1 <= numSvgChildren <= 3.